appveyor.yml

```diff
@@ -2,37 +2,44 @@
 #     DEVELOPMENT BRANCH
 #
 # build in Debug mode and deploy to MyGet.org as pre-release
-- version: 1.4.1-develop+{build}
-  branches:
+
+- branches:
     only:
     - develop
+
   configuration: Debug
   platform: Any CPU
+
   environment:
     use_build_number_in_packages: true
```

Comment:
also this could be safely removed, it is part of the old packaging mechanism on the common build script [appveyor-buildversion.ps1](https://github.com/RadicalFx/Radical.Build/blob/master/appveyor-buildversion.ps1)